build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
             "kotlin"         : "1.7.10",
             "androidxTest"   : "1.4.0",
             "room"           : "2.4.0",
-            "playServices"   : "19.2.1",
+            "playServices"   : "19.3.0",
```

Comment:
Do we need to call out any of the version bumps in this file or are they all internal?

Reply:
It doesn't look like we have been calling out any bumps in internal libraries.